pkg/chunked/storage_linux.go

```diff
@@ -1184,18 +1185,8 @@ func mergeMissingChunks(missingParts []missingPart, target int) []missingPart {
 		prev := missingParts[i-1].SourceChunk.Offset + missingParts[i-1].SourceChunk.Length
 		return int(missingParts[i].SourceChunk.Offset - prev)
```

Comment:
Unrelated, why it OK to down-cast to int(32) instead of keeping all offsets (u)int64?

(Also there seems to be only one caller now, with a `== 0`, so maybe we don’t need the subtraction in the first place.

Or, alternatively, use this also when computing the new `cost`, as `getGap(from, to)`)

Reply:
good observation.  It is safer to avoid the down-cast.  I'll change the type